scipy/stats/stats.py

```diff
@@ -5598,60 +5617,91 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate'):
 
     Notes
     -----
-    We can use this test, if we observe two independent samples from
-    the same or different population, e.g. exam scores of boys and
-    girls or of two ethnic groups. The test measures whether the
-    average (expected) value differs significantly across samples. If
-    we observe a large p-value, for example larger than 0.05 or 0.1,
-    then we cannot reject the null hypothesis of identical average scores.
-    If the p-value is smaller than the threshold, e.g. 1%, 5% or 10%,
-    then we reject the null hypothesis of equal averages.
+    Suppose we observe two independent samples, e.g. flower petal lengths, and
+    we are considering whether the two samples were drawn from the same
+    population (e.g. the same species of flower or two species with similar
+    petal characteristics) or two different populations.
+    The t-test quantifies the difference between the average (expected) values
+    of the samples, it and assesses the probability of such an observation
+    occuring by chance under the null hypothesis that the samples were drawn
+    from populations with the same average.
+    A p-value larger than our chosen threshold (e.g. 5% or 1%) indicates that
+    our observation is not so unlikely to have occured by chance, so we do
+    not reject the null hypothesis of identical averages.
+    If the p-value is smaller than our threshold, then we have evidence
+    against the null hypothesis of equal averages.
```

Comment:
```suggestion
    refuting the null hypothesis of equal population means.
```

Reply:
"refute" is a little strong, no?
"to _prove_ wrong by argument or evidence : show to be false or erroneous"
Not all definitions are "prove" but some are, and we wouldn't want to say that. Just a thought.

Reply:
yeah I prefer to avoid anything indicating 'proof', I usually prefer to reserve that for the mathematical object 'a proof of a theorem' to avoid ambiguity. Maybe then, 

```
against the null hypothesis of equal population means.
``` 
?

Reply:
Done. See in next commit.

Reply:
thanks